server.js

```diff
@@ -108,9 +117,9 @@ app.get('/come-back', (req, res) => {
 <body>
 <script>
     const jsReferrer = document.referrer;
-    document.body.innerHTML += '<p>header: <strong>${req.headers.referer}</strong></p><p>js: <strong>' + jsReferrer + '</strong></p>';
+    document.body.innerHTML += '<p>header: <strong>${req.headers.referer || ''}</strong></p><p>js: <strong>' + jsReferrer + '</strong></p>';
```

Comment:
unrelated referrer clean up - for Brave referrer is sometimes undefined in which scenarios we want to return `''` instead `'undefined'`